torchao/quantization/autoquant.py

```diff
@@ -433,20 +433,44 @@ def change_autoquantizable_to_quantized(model, **kwargs):
 # TODO: Document all the modes
 # TODO: Mode being a list is weird, should be a string or some object
 @torch.no_grad()
-def autoquant(model, example_input=None, qtensor_class_list=DEFAULT_CLASS_LIST, filter_fn=None, mode=["interpolate", .85], **aq_kwargs):
+def autoquant(
+    model, 
+    example_input=None, 
+    qtensor_class_list=DEFAULT_CLASS_LIST, 
+    filter_fn=None, mode=["interpolate", .85], 
+    manual=False, 
+    **aq_kwargs
+):
     """
-    Wraps the given model in an AutoQuantWrapper. If `example_input` is provided, performs a forward pass on the input.
-    Otherwise, returns the wrapped model. The AutoQuantWrapper manages cases where the model is torch-compiled by first
-    performing autoquantization on the original model and then allowing the torch.compile run/tracing to occur.
+    Autoquantization is a process which identifies the fastest way to quantize each layer of a model over some set of potential
```

Comment:
Could this helped quite a bit, can you also make sure it renders correctly here https://github.com/pytorch/ao/tree/main/docs/source